src/services/chris_integration.tsx

```diff
@@ -187,20 +184,22 @@ class ChrisIntegration {
         title: img.fname,
         imagefile: `${filename}.jpg`
       }
+
       if (covidnetPlugin === undefined || covidnetPlugin.data === undefined) {
         return [{
           plugin: pluginNeeded,
           error: new Error('not registered')
         }];
       }
       const covidnetInstance: PluginInstance = await client.createPluginInstance(covidnetPlugin.data.id, plcovidnet_data);
-      console.log("Covidnet Running");
+      console.log(`${pluginNeeded.toUpperCase()} task sent into the task queue`)
```

Comment:
String interpolation using backtick literals should only be used when necessary in my opinion. In this case, the plugin that gets run could be different depending on whether it's a CT scan or Xray scan.

In the case of `pl-dircopy` and `pl-med2img`, those are less likely to change. By using string interpolation when it's not necessary, it unnecessarily slows down the performance of the portal (which is negligible in smaller apps but as it grows, performance is important). String interpolation less performant because it requires more processing.